pythonProject/theory questions/exam.txt

```diff
@@ -0,0 +1,72 @@
+Questions:
+2-marks:
+1) What is Selenium?
+selenium is a web application automation framework.
+it has more tools and libraries to automate application
+
+2) Command to install selenium.
+pip install selenium
+3) What are the locators in Selenium?
+ID,name,classname,xpath,linked text,partially linked text,css selector
+4) What is the difference between find_element_by_* and find_elements_by_* in Selenium?
+find _element_by-*..... is used to find out the only single element by the locator
+ but ,find_element_by_* is used to find out the multiple element
+5) How do you manage browser windows in Selenium?
+driver.WindowHandles()
+driver.currentWindowHandle()
+switch_to()
+switch_to.(driver.window_handles(index))
+
+6) What are the frameworks used in selenium. Explain
```

Comment:
Why didn't we explain about POM.? And why we haven't shared anything technically. ?